### ISSUE TITLE
Unicorn Crash on Mac M1 / Should use upstream Unicorn

### ISSUE THREAD

**Issue by xhyumiracle** [ora-io/opml#11]
When running `` on Mac M1, it crashed with the same error msg described [here](https://github.com/unicorn-engine/unicorn/issues/1470#issuecomment-962653704).

According to the issue it should be fixed in this [commit](https://github.com/unicorn-engine/unicorn/commit/94a82ed94d266f33c802671ae140692835ddc80a).

Yet in opML we used a fork https://github.com/geohot/unicorn which doesn't include that fix, and updating the submodule itself doesn't simply fix it, since the Unicorn API version is different:

```
Different API version between core & binding (UC_ERR_VERSION)
```

In conclusion, I assume we'd better to adopt the upstream Unicorn version instead of the fork?
